src/Sulu/Component/Content/Document/Subscriber/WebspaceSubscriber.php

```diff
@@ -64,11 +63,7 @@ public function doHydrate(AbstractMappingEvent $event)
         $event->getAccessor()->set('webspaceName', $webspaceName);
     }
 
-    /**
-     * @param PersistEvent $event
-     */
     public function doPersist(PersistEvent $event)
```

Comment:
Yeah, it is thrown, but this subscriber should not listen to it anymore, should it?